apps/server/src/services/keyword/parseCsv.ts

```diff
@@ -0,0 +1,8 @@
+export const parseCsv = (csvFile: Express.Multer.File) => {
+  const contents = csvFile.buffer.toString('utf-8')
+  const keywords = contents.split(/\r?\n/).filter(Boolean)
+  if (keywords.length > 100) {
```

Comment:
It would be best not to hardcode such value (`100`) but instead define a constant or even an environment variables 💡